tests/model_selection/test_wrappers.py

```diff
@@ -0,0 +1,128 @@
+import pickle
```

Comment:
Can you split this into a `test_keras.py` and a `test_pytorch.py`?

Then you can use a `pytest.importorskip` at the top of the file and the tests won't have to worry about missing imports.